Documentation/ceph-advanced-configuration.md

```diff
@@ -590,3 +591,39 @@ ceph osd pool get replicapool crush_rule
 If the cluster's health was `HEALTH_OK` when we performed this change, immediately, the new rule is applied to the cluster transparently without service disruption.
 
 Exactly the same approach can be used to change from `host` back to `osd`.
+
+## Auto Expansion of OSDs
+
+### Prerequisites
+
+A [PVC-based cluster](ceph-cluster-crd.md#pvc-based-cluster) deployed in dynamic provisioning environment with a `storageClassDeviceSet`.
+  
+### To scale OSDs Vertically
+
+If you need to auto-grow the size of OSDs on a PVC-based Rook-Ceph cluster whenever the OSDs have reached the storage near-full threshold.
+
+You just need to run a script.
+```console
+tests/scripts/auto-grow-storage.sh size  --max maxSize --growth-rate percent 
+```
+>growth-rate percentage represents the percent increase you want in the OSD capacity and maxSize represent the maximum disk size.
+
+For example, If you need to increase the size of OSD by 30% and max disk size is 1Ti
+```console
+./auto-grow-storage.sh size  --max 1Ti --growth-rate 30 
+```
+
+### To scale OSDs Horizontally
+
+If you need to auto-grow the number of OSDs on a PVC-based Rook-Ceph cluster whenever the OSDs have reached the storage near-full threshold.
+
+You just need to run a script.
```

Comment:
Line 618 and 620 could be a single line - 
` Run the following script to auto-grow the number of OSDs ...`

Reply:
Done